frontend/src/components/ContentDisplayToggle.tsx

```diff
@@ -0,0 +1,32 @@
+import { Dispatch, SetStateAction } from "react";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+// this could implemented slightly differently to either be simpler and more generic
+// or we could opt to build the entire show / hide functionality into this component and pass the content controlled by the toggle as children
```

Comment:
USWDS has an accordion component for the more generic use case of toggling content display. This particular component needs to be more unique, so I'd name/build it with specificity — e.g. it's a toggle for the mobile sidebar layout.

Reply:
My hope was that this component could be reused in other scenarios, since all it's doing is displaying the toggle button and calling the toggle function that is passed in. If we don't want this to be a reusable component I can revert that though, and we could probably delete the story as well.